src/templates/schedule.html

```diff
@@ -10,6 +10,7 @@
   <ul>
     <li>Room 1 Lynnhaven <a href="http://dominionenterprises.adobeconnect.com/devcon1">http://dominionenterprises.adobeconnect.com/devcon1</a></li>
     <li>Room 2 York <a href="http://dominionenterprises.adobeconnect.com/devcon2">http://dominionenterprises.adobeconnect.com/devcon2</a></li>
+    <li>Room 3 Rappahannock <a href="http://dominionenterprises.adobeconnect.com/devcon3">http://dominionenterprises.adobeconnect.com/devcon3</a></li>
```

Comment:
The fourth room if used at all will not a media site link so I don't think it needs to be mentioned here.